Add a --disable-tools option

This patch adds a --disable-tools option that allows to disable the
compilation of the gettext tools, which may not be necessary on a
small embedded Linux system.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>

Index: b/configure.ac
===================================================================
--- a/configure.ac
+++ b/configure.ac
@@ -37,6 +37,10 @@
 
 dnl Checks for library functions.
 
+AC_ARG_ENABLE([tools], [--disable-tools  do not build tools],
+		       [enable_tools=$enableval], [enable_tools=yes])
+AM_CONDITIONAL([ENABLE_TOOLS], [test "$enable_tools" = "yes"])
+
 AC_CONFIG_SUBDIRS([gettext-runtime gettext-tools])
 
 dnl Ensure that "configure --help" lists all the command line options that
Index: b/Makefile.am
===================================================================
--- a/Makefile.am
+++ b/Makefile.am
@@ -20,7 +20,13 @@
 ACLOCAL = build-aux/fixaclocal @ACLOCAL@
 ACLOCAL_AMFLAGS = -I m4
 
-SUBDIRS = gnulib-local gettext-runtime gettext-tools
+if ENABLE_TOOLS
+TOOLS_SUBDIR = gettext-tools
+else
+TOOLS_SUBDIR =
+endif
+
+SUBDIRS = gnulib-local gettext-runtime $(TOOLS_SUBDIR)
 
 EXTRA_DIST = \
   version.sh DEPENDENCIES PACKAGING HACKING ChangeLog.0 autogen.sh \
